<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VRCWHostProjectionClass</class>
 <widget class="QWidget" name="VRCWHostProjectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VRCWHostProjection</string>
  </property>
  <layout class="QVBoxLayout" name="vrcwHostProjectonClassVLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>66</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="headerWidgetHLayout">
      <item>
       <layout class="QHBoxLayout" name="headerHLayout">
        <item>
         <layout class="QVBoxLayout" name="headingVLayout">
          <item>
           <widget class="QLabel" name="header">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Host Projection Configuration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="description">
            <property name="text">
             <string>Assign hosts to projection</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="headerVSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="headerHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="headerLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QGridLayout" name="mainWidgetGLayout">
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QWidget" name="coviseGuiHostWidget" native="true">
        <layout class="QHBoxLayout" name="guiMasterRendererWidgetHLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="coviseGuiHostLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>COVISE GUI host</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="coviseGuiHostComboBox">
           <property name="toolTip">
            <string>The host on which you start the COVISE GUI</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="minimumContentsLength">
            <number>14</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignLeft">
       <widget class="QWidget" name="hostProjectionWidget" native="true">
        <layout class="QHBoxLayout" name="hostProjectionWidgetHLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="projectionLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Projection</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="projectionComboBox">
           <property name="insertPolicy">
            <enum>QComboBox::InsertAlphabetically</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="minimumContentsLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hostProjectionHSpacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="hostLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="hostComboBox">
           <property name="insertPolicy">
            <enum>QComboBox::InsertAlphabetically</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="minimumContentsLength">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hostProjectionHSpacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="hostProjectionAddPushButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignLeft">
       <widget class="QWidget" name="tableWidget" native="true">
        <layout class="QHBoxLayout" name="tableWidgetHLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="tableView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>250</height>
            </size>
           </property>
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="tableButtonVLayout">
           <property name="topMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="QPushButton" name="tableRemovePushButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="tableButtonVSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>hostProjectionAddPushButton</sender>
   <signal>clicked()</signal>
   <receiver>VRCWHostProjectionClass</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hostProjectionAddPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tableView</receiver>
   <slot>resizeColumnsToContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableRemovePushButton</sender>
   <signal>clicked()</signal>
   <receiver>VRCWHostProjectionClass</receiver>
   <slot>remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableRemovePushButton</sender>
   <signal>clicked()</signal>
   <receiver>tableView</receiver>
   <slot>resizeColumnsToContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>remove()</slot>
 </slots>
</ui>
